<ui version="4.0" >
 <class>PickBinDialogBase</class>
 <widget class="QDialog" name="PickBinDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pick Binnings from list...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" name="hBoxLayoutTop" >
     <item>
      <widget class="QLabel" name="TextLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Pick your list of binnings...or hold the 'Ctrl' key for multiple selections, or hold the 'Shift' key for a range selection.</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vBoxLayoutTop" >
       <item>
        <widget class="QPushButton" name="PushButtonOk" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonCancel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>59</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Bins available</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="treeWidget" >
        <property name="statusTip" >
         <string>Pick your bins from the list...</string>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Soft Bin</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>SBIN name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Hard Bin</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>HBIN name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
